script/deploy/healthCheck.ts

```diff
@@ -506,7 +506,12 @@ const main = defineCommand({
         const safeAddress = networkConfig.safeAddress
         const safeApiUrl = networkConfig.safeApiUrl
         const configUrl = `${safeApiUrl}/v1/safes/${safeAddress}`
-        const res = await fetch(configUrl)
+        let res
+        try {
+          res = await fetch(configUrl)
+        } catch (error) {
+          logError(`Could not fetch SAFE config from configURL: ${configUrl}`)
+        }
         const safeConfig = await res.json()
```

Comment:
How is this supposed to work? If the fetch call fails and an error is logged, the next line will also fail. Is the catch only in place for the logging or should it prevent that the whole script fails?

Reply:
The script should fail in this case but you are right, this could be made more explicit by exiting with an error code in the catch clause. Will update it.